mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -540,7 +523,14 @@ def _create_tables_if_not_exist(self):
         for table in self._tables:
             # Create table if not exist. The `metadata` contains the `ModelEndpointsTable`
             if not self._engine.has_table(table):
+                logger.info(
+                    f"Creating table {table} on {self._sql_connection_string.split('/')[-1]} db."
```

Comment:
I guess it makes sense to extract `self._sql_connection_string.split('/')[-1]` outside the if condition and for loop.